projects/ziti-console-lib/src/lib/features/projectable-forms/service/service-form.component.ts

```diff
@@ -177,10 +174,54 @@ export class ServiceFormComponent extends ProjectableForm implements OnInit, OnC
     this.svc.configChanged();
   }
 
+  configFilterChanged(event) {
+    if (event?.keyCode === 37 || event?.keyCode === 38 || event?.keyCode === 39 || event?.keyCode === 40) {
```

Comment:
can these be helpful constants of some kind? LEFT_ARROW, RIGHT_ARROW etc?

Reply:
Yep I'll add constant for known/expected key_codes